potools.Rproj

```diff
@@ -18,3 +18,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd
```

Comment:
Could you elaborate what this does? Happy to add it, just don't see anything in a quick search

Reply:
This just tell RStudio to use roxygen2 only for .Rd files and not the namespace.